schema/input_fields/partner_classification.js

```diff
@@ -0,0 +1,41 @@
+import { GraphQLEnumType } from 'graphql';
+
+export default {
+  type: new GraphQLEnumType({
+    name: 'PartnerClassification',
+    values: {
+      AUCTION: {
+        value: 'PartnerAuction',
+      },
+
+      DEMO: {
+        value: 'PartnerDemo',
+      },
+
+      GALLERY: {
+        value: 'PartnerGallery',
+      },
+
+      PRIVATE_COLLECTOR: {
+        value: 'PartnerPrivateCollector',
+      },
+
+      PRIVATE_DEALER: {
+        value: 'PartnerPrivateDealer',
+      },
+
+      INSTITUTION: {
+        value: 'PartnerInstitution',
+      },
+
+      INSTITUTIONAL_SELLER: {
+        value: 'PartnerInstitutionalSeller',
+      },
+
+      BRAND: {
+        value: 'PartnerBrand',
+      },
+
```

Comment:
Vaguely surprised eslint didn't yell about this whitespace. Though I think the task has --fix on it which it shouldn't.